tests/unit/routes/test_person_router.py

```diff
@@ -112,3 +117,74 @@ def test_with_patients(self, client):
             "person_reference_id": str(person.reference_id),
             "patient_reference_ids": [str(pat1.reference_id), str(pat2.reference_id)],
         }
+
+
+class TestMergePersonClusters:
+    def testMergePersonClustersSuccess(self, client):
+        person1 = models.Person()
+        patient1 = models.Patient(person=person1, data={})
+
+        person2 = models.Person()
+        patient2 = models.Patient(person=person2, data={})
+
+        client.session.add_all([patient1, patient2])
+        client.session.flush()
+
+        response = client.post(
+            f"/person/{person1.reference_id}/merge/",
+            json={"person_reference_ids": [str(person2.reference_id)]},
+        )
+
+        assert response.status_code == 200
+        assert response.json()["person_reference_id"] == str(person1.reference_id)
+
+    def testInvalidPersonIdType(self, client):
+        response = client.post(
+            "/person/123/merge/",
+            json={"person_reference_ids": [str(uuid.uuid4())]},
+        )
+        assert response.status_code == 422
+
+    def test1InvalidPersonId(self, client):
```

Comment:
Could we add a test case to show that the person ids specified in the body of the payload can't be the same as the one specified in the URL?